cmake/DetectCodeCoverage.cmake

```diff
@@ -20,7 +26,8 @@ if ( GCOV AND
      LCOV AND
```

Comment:
The three blocks starting at https://github.com/quinoacomputing/quinoa/blob/master/script/update_doc.sh#L59
shows how we generate coverage currently.